debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java

```diff
@@ -568,7 +568,11 @@ public StructureDescriptor(String line) {
 		}
 
 		public String toString() {
-			return name + " extends " + superName;
+			if (superName == null || superName.isEmpty()) {
+				return name;
```

Comment:
For compability, this should be
```
  return String.valueOf(name);
```
to avoid the possibility of a null return value (which could not previously occur).